components/security/src/lib.rs

```diff
@@ -152,10 +153,20 @@ impl SecurityManager {
         }
     }
 
-    pub fn bind(&self, sb: ServerBuilder, addr: &str, port: u16) -> ServerBuilder {
+    pub fn bind(&self, mut sb: ServerBuilder, addr: &str, port: u16) -> ServerBuilder {
         if self.cfg.ca_path.is_empty() {
             sb.bind(addr, port)
         } else {
+            if !self.cfg.cert_allowed_cn.is_empty() {
+                let cert_allowed_cn = self.cfg.cert_allowed_cn.clone();
+                sb = sb.add_checker(move |ctx| match check_common_name(&cert_allowed_cn, ctx) {
+                    Ok(()) => CheckResult::Continue,
+                    Err(reason) => CheckResult::Abort(RpcStatus::new(
+                        RpcStatusCode::UNAUTHENTICATED,
+                        Some(format!("Failed to check common name, reason: {}", reason)),
```

Comment:
```suggestion
                        Some(format!("Common name check fail, reason: {}, cert_allowed_cn: {}", reason, cert_allowed_cn)),
```

Reply:
bringing this into the log may incur insecurity?

Reply:
IIUC common name is not a secret?

Reply:
Got